posthog/hogql/test/test_printer.py

```diff
@@ -134,6 +134,29 @@ def test_hogql_properties(self):
             "properties.'no strings'", "mismatched input ''no strings'' expecting DECIMAL_LITERAL", "hogql"
         )
 
+    def test_hogql_properties_json(self):
+        try:
+            from ee.clickhouse.materialized_columns.analyze import materialize
+        except:
+            # EE not available? Assume we're good
```

Comment:
Seems a bit hacky to check for EE availability like this. The test could become a noop when we move things around.

Reply:
Yeah, EE tests just need to be in `ee/` rather than `posthog/`. Hard to trust a test that self-parametrizes itself this way.